### PR TITLE
koord-scheduler: improve the readability of the PreFilter error message of the DeviceShare plugin

### LINKED ISSUE #840: [proposal] Improve the readability of the PreFilter error message of the DeviceShare plugin
**What is your proposal**:

When I created a Pod with resource requests `kubernetes.io/gpu-core=50`, `kubernetes.io/gpu-memory=2Gi`  and  `kubernetes.io/gpu-memory-ratio=50`, I got the PreFilter error like this: `request is not valid, current combination: 010001000` .

This error message cannot be understood intuitively.

**Why is this needed**:

make sense of error messages.

**Is there a suggested solution, if so, please add it**:

### PR DESCRIPTION
**Pull request by jasonliu747**
### Ⅰ. Describe what this PR does
Currently, the error message of PreFiltre in DeviceShare plugin cannot be understood intuitively.
This PR will improve the readability.

### Ⅱ. Does this pull request fix one issue?
fix #840

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/scheduler/plugins/deviceshare/utils.go [hunk 1]
```diff
@@ -117,7 +117,7 @@ var ValidateGPURequest = func(podRequest corev1.ResourceList) (uint, error) {
 		return gpuCombination, nil
 	}
 
-	return gpuCombination, fmt.Errorf("request is not valid, current combination: %b", gpuCombination)
+	return gpuCombination, fmt.Errorf("request is not valid, current combination: %v", convertGPURequestCombination(gpuCombination))
```
Comment (ZiMengSheng) [inline pkg/scheduler/plugins/deviceshare/utils.go:120]: maybe we can give what is right?

Comment (jasonliu747) [reply pkg/scheduler/plugins/deviceshare/utils.go]: image

Comment (eahydra) [reply pkg/scheduler/plugins/deviceshare/utils.go]: hmm.. we should update the comments..

**Pull request merged**